app/lib/actions/remote_execution/run_host_job.rb

```diff
@@ -24,11 +24,16 @@ def plan(job_invocation, host, template_invocation, proxy)
 
         raise _('Could not use any template used in the job invocation') if template_invocation.blank?
 
-        settings = { :global_proxy   => 'remote_execution_global_proxy',
-                     :fallback_proxy => 'remote_execution_fallback_proxy' }
+        if proxy.blank?
+          offline_count = options[:offline_count] || 0
+          raise N_('All %{number} usable proxies are down.') % { :number => offline_count } if offline_count > 0
```

Comment:
Ah I meant lower case `n`, it handles plurals, so you can give:

`n("message when there's one", "message when > 1", count)`

Here's an example:

https://github.com/theforeman/foreman/blob/develop/app/helpers/layout_helper.rb#L485

Otherwise the way it was would render "All 1 usable proxies are down" which isn't correct English (and probably not correct in a bunch of languages).  It should read either "The only usable proxy is down" or "All X usuable proxies are down."